### PR TITLE
Shell0002/tutorials

### PR DESCRIPTION
**Pull request by shellygill**
@TADraeseke please review this Kotlin samples update. Mainly this is an update to include the Display map sample, so it can be referenced from new Kotlin first map app tutorial (includes code snippets).

Additionally:
- updated kotlin version
- updated dependency of kotlin now that the kotlin libraries and extensions are released
- fixed the existing authentication sample - logic was broken, sample was not working

### CODE REVIEW

File: kotlin/display-map/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.esri.arcgisruntime.sample.displaymap">
+
```
Comment (TADraeseke) [inline kotlin/display-map/src/main/AndroidManifest.xml]: Needs internet permission

File: kotlin/display-map/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:aapt="http://schemas.android.com/aapt"
+        android:width="108dp"
+        android:height="108dp"
+        android:viewportHeight="108"
+        android:viewportWidth="108">
+  <path
```
Comment (TADraeseke) [inline kotlin/display-map/src/main/res/drawable-v24/ic_launcher_foreground.xml]: I don't know what these two launcher xml files do, but I assume they're intended (and important!) :)

File: kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt [hunk 3]
```diff
@@ -43,8 +43,10 @@ class MainActivity : AppCompatActivity() {
         val portal = Portal("http://www.arcgis.com", true)
```
Comment (TADraeseke) [inline kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt]: Fix to logic makes sense to me. Just a heads up that I get 'Portal failed to load' toast when I try running the authentication profile app

File: kotlin/display-map/src/main/res/drawable-v24/ic_launcher_foreground.xml (hunk 2 above)
Comment (shellygill) [reply kotlin/display-map/src/main/res/drawable-v24/ic_launcher_foreground.xml]: Adaptive drawables for the launcher icons - default for kotlin app template with latest AS.

File: kotlin/display-map/src/main/AndroidManifest.xml (hunk 1 above)
Comment (shellygill) [reply kotlin/display-map/src/main/AndroidManifest.xml]: Fixed, not sure how I missed that!

File: kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt (hunk 3 above)
Comment (shellygill) [reply kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt]: This is working OK for me with latest daily build. Can you repro the same issue with the Java version, same build?

**Comment by shellygill** [comment]
Thanks @TADraeseke - fix pushed.

File: kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt (hunk 3 above)
Comment (TADraeseke) [reply kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt]: Ok, its working for me now on the latest build. 'Portal failed to load' was on build 1890.

The formatting is different for me between kotlin and java.

**Comment by TADraeseke** [comment]
@shellygill Good to merge from me, but just a note that there is a difference in formatting between java and kotlin

**Pull request merged**